tests/test_skills/test_abstract_round_abci/test_base.py

```diff
@@ -1016,6 +1017,8 @@ def test_get_earliest_timeout_a(self) -> None:
         deadline_1 = datetime.datetime.now()
         event_1 = MagicMock()
 
+        sleep(0.5)
```

Comment:
Looks like python3.10 is faster on windows compared to other OS, since deadline_1 and deadline_2 end up being same so I added this tiny bit of delay